Makefile

```diff
@@ -10,7 +10,7 @@ SELF:=$(firstword $(MAKEFILE_LIST))
 
 PKG=hgvs
 PKGD=$(subst .,/,${PKG})
-TEST_DIRS:=.,doc,hgvs,tests
```

Comment:
Thanks @icebert, that's very helpful. I updated the Makefile to look for doctests in RST files in 184de2578c0bc8d45cef38a5d60994d36e8225cd. Let me know if you want me to squash that commit.

Including `.`, pytest was reporting errors collecting tests from `setup.py`, and some of the example files in `misc/` which didn't work correctly in the virtualenv (no module `tabulate`, etc.). As far as I can tell, there are no tests in misc, so I think excluding it is fine.